### DISCUSSION TITLE
Is there a way to show traffic on the map using this module?

### DISCUSSION THREAD

**Discussion by 7ur813z** [rnmapbox/maps#3539] [Q&A]
Does this module include the Mapbox Traffic functionality? I found another module that does include it. It is called Mapbox-Gl-Traffic, but it's for React not React Native. 

- Mapbox Traffic: https://docs.mapbox.com/data/traffic/guides/ 
- Mapbox-Gl-Traffic: https://github.com/mapbox/mapbox-gl-traffic

**Comment by mfazekas** [comment]
You should be able to display the traffic vector source, with a line layer for example - https://docs.mapbox.com/data/tilesets/reference/mapbox-traffic-v1/
